src/tools/qt-gui/src/treeviewmodel.cpp

```diff
@@ -791,6 +792,34 @@ void TreeViewModel::showConfigNodeMessage (QString title, QString text, QString
 	emit showMessage (title, text, detailedText);
 }
 
+void TreeViewModel::connectDBus()
+{
+    if ( QDBusConnection::sessionBus().connect( QString(), "/org/libelektra/configuration", "org.libelektra", QString(), this, SLOT( configChanged( QString ) ) ) )
+    {
+        #if DEBUG && VERBOSE
```

Comment:
Please use `ELEKTRA_LOG("Successfully connected to DBus");` here (needs `#include kdblogger.h`). To see the output set ELEKTRA_LOG for cmake (and you need to rebase to version 01866af01fde59ff3ca6bb1cf55d35e89c38a9c9).